azure-pipelines.yml

```diff
@@ -61,7 +60,7 @@ jobs:
     steps:
       - task: NodeTool@0
         inputs:
-          versionSpec: '10.x'
+          versionSpec: '12.x'
```

Comment:
aren't we up to node 14 or is this the highest allowed?

Reply:
Yes, we are up to 14. I have been using node 12. I'll update to 14

Reply:
The environment uses version 12